pkgs/http/lib/src/browser_client.dart

```diff
@@ -53,6 +64,11 @@ class BrowserClient extends BaseClient {
 
   bool _isClosed = false;
 
+  /// Use different caching mode for a HTTP request.
+  /// Defaults to `default`.
+  // https://developer.mozilla.org/en-US/docs/Web/API/Request/cache
```

Comment:
I'd ask this NOT be buried in a normal comment. A doc comment so folks can go see the supported values would be GREAT!

Reply:
I see triple slashes /// would go rather than normal comment but, I try to pass the dart analyze so, I would be sure that the quality comes first.